native/common/include/jp_method.h

```diff
@@ -26,11 +26,7 @@ public :
 	 * Create a new method based on class and a name;
 	 */
 	JPMethod(jclass clazz, const string& name, bool isConstructor);
-	virtual ~JPMethod()

```

Comment:
Having implementations of virtual methods though legal is usually a bad practice.  The complier will have to build all those implementations and then the linker has to drop the spares.